Pod/Classes/WPMediaPickerOptions.h

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+#import "WPMediaCollectionDataSource.h"
+
+@interface WPMediaPickerOptions: NSObject
+
+/**
+ If set the picker will show a cell that allows capture of new media, that can be used immediatelly
+ */
+@property (nonatomic, assign) BOOL allowCaptureOfMedia;
```

Comment:
For each of these BOOL properties, I'd say "If YES" not "if set", just because it's a bit more explicit. You could set it to `NO`, after all.